src/main/java/com/mcmoddev/mmdbot/modules/commands/community/contextmenu/message/ContextMenuGist.java

```diff
@@ -95,11 +96,16 @@ public static String generateName(int length) {
             .toString();
     }
 
+    public static final List<String> BLACKLISTED_ATTACHEMENTS = List.of("png", "jpg", "jpeg", "webm");
```

Comment:
Probably worth cross-referencing the list of MAME multimedia types here - HEVC and mp4 will probably show up a lot in MMD.